plugin.js

```diff
@@ -25,6 +25,19 @@ function fastifyCookieSetCookie (reply, name, value, options, signer) {
   }
 
   const serialized = cookie.serialize(name, value, opts)
+
+  if (opts.enableWarnOnSafeLimit === true) {
```

Comment:
I would just go with `if (opts.warnOnSafeLimit > 0)` as suggested before instead of bool+integer